http/headers/feature-policy.json

```diff
@@ -1319,6 +1319,55 @@
             }
           }
         },
+        "speaker-selection": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/speaker-selection",
+            "spec_url": "https://w3c.github.io/mediacapture-output/#permissions-policy-integration",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "92"
+              },
+              "firefox_android": {
+                "version_added": "92"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": "14.0"
+              },
+              "webview_android": {
+                "version_added": "84"
+              }
```

Comment:
These values look suspect, based on the other data here and your PR description. Is this a copy and paste glitch, maybe?

Reply:
You're right - sloppy. Fixed.